essence-of-live-coding-frp-test/app/Main.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE Arrows #-}
+
+{- ORMOLU_DISABLE -}
+
+-- base
+import GHC.Float (double2Float)
+
+-- transformers
+import Control.Monad.Trans.Reader ( ask )
+
+-- gloss
+import Graphics.Gloss (Picture, color, green, thickCircle)
+import qualified Graphics.Gloss as Gloss
+
+-- essence-of-live-coding
+import LiveCoding
+  ( Arrow (arr),
+    Cell,
+    HandlingStateT,
+    LiveProgram,
+    arrM,
+    foreverE,
+    liftCell,
+    liveCell,
+    liveMain,
+    returnA,
+    (>>>),
```

Comment:
Hah, I didn't know this is valid syntax

Reply:
I generate these lists by accepting hlint suggestions is vscode and then formatted with ormolu. I read somewhere that, if possible, explicit import lists are better because if a package in a new version adds a new function that clashes with another existing function that will not cause a problem.